katello/katello/hostname-change.rb

```diff
@@ -256,8 +256,24 @@ def run
 
       public_dir = "/var/www/html/pub"
       public_backup_dir = "#{public_dir}/#{@old_hostname}-#{self.timestamp}.backup"
-      STDOUT.puts "deleting old certs"
+      
+      STDOUT.puts "removing old cert rpms"
+      cert_packages = [
+        "foreman-proxy-client*",
```

Comment:
The stars are not needed since that's done in the map. Right now it ends up as ```HOSTNAME-foreman-proxy-client**```. Could you change this? I generally also like sorted lists.